composer.json

```diff
@@ -17,7 +17,8 @@
         "phpcr/phpcr": "~2.1.1",
         "phpcr/phpcr-implementation": "~2.1.0",
         "phpcr/phpcr-utils": "~1.1",
-        "doctrine/instantiator": "~1.0.1"
+        "doctrine/instantiator": "~1.0.1",
+        "jackalope/jackalope-jackrabbit": "~1.0"
```

Comment:
please remove this from the composer.json again - you indeed need to do this to develop, but we don't want to lock phpcr-odm to a specific phpcr implementation.